holmes/clients/robusta_client.py

```
@@ -41,6 +42,24 @@ class RobustaModelsResponse(BaseModel):
+
+    params = {
+        "account_id": account_id,
+        "cluster": cluster,
+        "component": "holmes",
+        "component_version": get_version(),
+    }
+    try:
+        response = requests.get(SUPABASE_KEYS_URL, params=params, timeout=10)
+        response.raise_for_status()
+        return response.json().get("api_key")
+    except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)